bentoml/yatai/web/src/server.ts

```diff
@@ -220,20 +225,29 @@ export const getExpressApp = (grpcAddress: string | null, baseURL: string) => {
   );
   const yataiClient = createYataiClient(grpcAddress);
   const router = createAPIRoutes(app, yataiClient);
-  const apiUrlPrefix = (baseURL=="." ? '/' : '/' + baseURL);
+  const apiUrlPrefix = baseURL == "." ? "/" : "/" + baseURL;
   app.use(apiUrlPrefix, router);
 
-  app.get('/healthz', (req, res) => res.status(200).json());
+  app.get("/healthz", (req, res) => res.status(200).json());
+
+  app.get("/metrics", async (req, res) => {
+    const metricsResponse = await axios.get(prometheusAddress);
```

Comment:
instead of fetching it via prometheusAddress, could we just add another gRPC endpoint to expose the metrics?

Reply:
I can work on this if you think add gRPC endpoints is better than the current implementation?

Reply:
I think either way is fine, no need to make that change, unless @yubozhao or others think differently